web/packages/core/src/ruffle-player.js

```diff
@@ -152,15 +169,25 @@ exports.RufflePlayer = class RufflePlayer extends HTMLElement {
      * being loaded, or any errors that happen loading it.
      *
      * @param {String} url The URL to stream.
+     * @param {URLSearchParams|String|Object} [parameters] The parameters (also known as "flashvars") to load the movie with.
+     * If it's a string, it will be decoded into an object.
+     * If it's an object, every key and value must be a String.
+     * If it's null or undefined, it will attempt to be parsed from the URL. If you explicitly want no flashvars, use {}.
      */
-    async stream_swf_url(url) {
+    async stream_swf_url(url, parameters) {
```

Comment:
I wonder if there should be a RufflePlayer.flashVars setter to set this? Although I guess you can't change it after the file loads, so having it as a parameter here makes sense.

Reply:
I felt like this made more sense for an API. Having it be a property or method on the player suggests that they can be changed at will.